accessibility-oscilloscope/configs/rpi4_defconfig

```diff
@@ -34,3 +34,6 @@ BR2_PACKAGE_ASCII_CONV=y
 BR2_PACKAGE_DRV2506_CLEAN=y
 BR2_PACKAGE_PNT_LUT=y
 BR2_PACKAGE_SIGNAL_COMPILER=y
+BR2_PACKAGE_SIGPROC=y
+BR2_PACKAGE_OSCILLOSCOPE_TRIGGER=y
```

Comment:
at this point we have everything packaged right? if so i'll make an issue to make sure everything is enabled for each `DEFCONFIG`.

Reply:
Yes I believe so